wdl/meta.wdl

```diff
@@ -1,27 +1,317 @@
-import "meta.sub.wdl" as sub
+version 1.0
 
 workflow meta_analysis {
 
-    File sumstats_loc
-    File pheno_confs
+    input {
+        File sumstats_loc
+        File pheno_confs
 
-    Array[Array[String]] sumstat_files = read_tsv(sumstats_loc)
-    Array[String] pheno_conf = read_lines(pheno_confs)
+        Array[Array[String]] sumstat_files = read_tsv(sumstats_loc)
+        Array[String] pheno_conf = read_lines(pheno_confs)
+    }
 
     scatter (i in range(length(pheno_conf))) {
 
         String pheno = basename(pheno_conf[i], ".json")
+        
+        scatter (chr in range(23)) {
```

Comment:
oh my effin gooood!!! 
does this actually work and how is the output organized?